todolist copy 3/main.js

```diff
@@ -0,0 +1,103 @@
+"use strict";
+
+const addTaskButton = document.getElementById('addTask');
+const lineTaskInput = document.getElementById('taskInput');
+const toDoList = document.getElementById('tasklist');
+const clearing = document.getElementById('clearAll');
+const checkBoxCover = document.getElementById('clearAll');
+
+
+let tasks = JSON.parse(localStorage.getItem("tasks")) ?? [];
+!localStorage.tasks ? tasks = [] : tasks = JSON.parse(localStorage.getItem('tasks'));
+
+function Task(description) {
+	this.description = description;
+	this.completed = false
+}
+
+let toDoElem = [];
+
+const updateHTML = () => {

```

Comment:
a function should be responsible for only one thing, so if it's updateHTML (better to call it updateList) then it means that the function should do only this one job
That's why we need to extract createCheckbox, createDeleteButton, createText, deleteTask, createTodo into separate functions

in createTodo we will take all elements that we've created in other functions and append them to li